yt_dlp/options.py

```diff
@@ -515,6 +515,17 @@ def _alias_callback(option, opt_str, value, parser, opts, nargs):
         metavar='IP', dest='source_address', default=None,
         help='Client-side IP address to bind to',
     )
+    network.add_option(
+        '--impersonate',
+        metavar='CLIENT[:[VERSION][:OS[:OS_VERSION]]]', dest='impersonate', default=None,
+        help='Client to impersonate for requests. E.g. chrome, chrome:110, chrome::android.'
+             ' Pass in an empty string (--impersonate="") to impersonate any client.',
```

Comment:
To conform with the other options formatting
```suggestion
        help=(
            'Client to impersonate for requests. E.g. chrome, chrome:110, chrome::android. '
            'Pass in an empty string (--impersonate="") to impersonate any client.'),
```